test/distributed/org/apache/cassandra/distributed/test/SSTableLoaderEncryptionOptionsTest.java

```diff
@@ -49,16 +49,16 @@
 
 public class SSTableLoaderEncryptionOptionsTest extends AbstractEncryptionOptionsImpl
 {
-    static Cluster CLUSTER;
-    static String NODES;
-    static int NATIVE_PORT;
-    static int STORAGE_PORT;
-    static int SSL_STORAGE_PORT;
+    static Cluster cluster;
```

Comment:
@Runtian I think you fixed the wrong test.

Reply:
oops, let me fix this.